client/src/main/proto/spine/client/query.proto

```diff
@@ -40,13 +40,20 @@ import "spine/client/entities.proto";
 message QueryContext {
 
     ui.Language language = 1;
+
     //TODO:2015-12-16:alexander.yevsyukov: Finish
```

Comment:
No. The comments have what needs to be done in order to make this type definition feature complete for v1.